src/NzbDrone.Core/Indexers/BroadcastheNet/BroadcastheNetSettings.cs

```diff
@@ -40,6 +44,9 @@ public BroadcastheNetSettings()
         [FieldDefinition(4, Type = FieldType.Checkbox, Label = "IndexerSettingsRejectBlocklistedTorrentHashes", HelpText = "IndexerSettingsRejectBlocklistedTorrentHashesHelpText", Advanced = true)]
         public bool RejectBlocklistedTorrentHashesWhileGrabbing { get; set; }
 
+        [FieldDefinition(5, Type = FieldType.Select, SelectOptions = typeof(RealLanguageFieldConverter), Label = "Multi Languages", HelpText = "What languages are normally in a multi release on this indexer?", Advanced = true)]
```

Comment:
These need to be added to `en.json` and the key for the translations should be used here (and other instances of settings) instead.